editoast/src/views/train_schedule.rs

```diff
@@ -95,6 +96,9 @@ pub enum TrainScheduleError {
     #[error("{number} train schedule(s) could not be found")]
     #[editoast_error(status = 404)]
     BatchTrainScheduleNotFound { number: usize },
+    #[error("{number} rollings stock(s) could not be found")]
+    #[editoast_error(status = 404)]
+    BatchRollingStockNotFound { number: usize },
```

Comment:
We should not return an error in this case. The previous behavior was the right one.

If a RS is not found in the batch we should still compute all the simulations. Trains that don't have a valid RS are marked as invalid.